torch/_inductor/codecache.py

```diff
@@ -917,6 +917,10 @@ def install_gcc_via_conda() -> str:
 
 
 def is_gcc() -> bool:
+    # Mac OS not support real gcc, it is acturally a clang.
+    _IS_MACOS = sys.platform.startswith("darwin")
+    if _IS_MACOS:
+        return False
```

Comment:
Why not that, to be aligned with other similar checks in that file?
```suggestion
    if sys.platform == "darwin":
        return False
```